benchmark/runner/tpch_runner.cpp

```diff
@@ -20,6 +20,9 @@ class TPCHRunner : public benchmark::Fixture {
   const std::string tpch_table_root_ = "../../../tpl_tables/tables/";
   const std::string tpch_database_name_ = "tpch_db";
```

Comment:
It's not only tpch anymore so maybe we should call this `database_name_` with something like "benchmark_db".